autogpt/core/ability/schema.py

```diff
@@ -1,10 +1,21 @@
-from typing import Any
+import enum
+from typing import Any, Union
+
 from pydantic import BaseModel
 
 
+class ContentType(str, enum.Enum):
+    # TBD what these actually are.
+    TEXT = "text"
+    CODE = "code"
+    IMAGE = "image"
+    VIDEO = "video"
+    AUDIO = "audio"
+
+
 class Knowledge(BaseModel):
     content: str
```

Comment:
nit: May want to make this str|path or specify what the format of video, image and audio is allowed. 

Not a huge issue but could help when we move to multimodal implementations

Reply:
Good thought.  Trying not to over-engineer before we've got use cases, but will keep in mind.